src/calendar/header/index.js

```diff
@@ -102,7 +103,8 @@ class CalendarHeader extends Component {
     const {disabledDaysIndexes} = this.props;
     return weekDaysNames.map((day, idx) => {
       const dayStyle = [this.style.dayHeader];
-      if (disabledDaysIndexes && disabledDaysIndexes.length) {
+
+      if (!_.isEmpty(disabledDaysIndexes)) {
         if (disabledDaysIndexes.includes(idx)) {
```

Comment:
Sorry for nagging on this..
but I believe it can be even cleaner 
```suggestion
      if (_.includes(disabledDaysIndexes, idx)) {
```